frontend/apps/filemanager/filemanager.lua

```diff
@@ -272,6 +272,16 @@ function FileManager:setupLayout()
                     file_manager.collections:genAddToCollectionButton(file, close_dialog_callback, refresh_callback),
                 })
             end
+            if Device:canExecuteScript(file) then
+                table.insert(buttons, {
+                    filemanagerutil.genExecuteScriptButton(file, close_dialog_callback),
+                })
+            end
+            if FileManagerConverter:isSupported(file) then
+                table.insert(buttons, {
+                    FileManagerConverter:genConvertButton(file, close_dialog_callback, refresh_callback)
+                })
+            end
```

Comment:
This being moved around will just insert full rows above instead of below - no change in the layout of the other buttons?

Reply:
Please look at the screenshot.
Currently 'Convert' and 'Execute script' full-width buttons are added between 'Book cover' and 'Ignore cover' rows (even if we remove providers for oy and sh, the provider for md should remain).
I move Convert and Execute buttons upper, without changing the layout of other buttons.

Reply:
To keep related 'cover' buttons together.

Reply:
Ok, that's fine.

> Please look at the screenshot.

(I didn't see any of these rarely-seen buttons on your screenshot, so that's why I asked.)